common/tvm_utility/include/tvm_utility/pipeline.hpp

```diff
@@ -224,6 +224,76 @@ typedef struct
 class InferenceEngineTVM : public InferenceEngine
 {
 public:
+  explicit InferenceEngineTVM(
```

Comment:
Please don't duplicate the code. What you could do is have `autoware_data_path` as a parameter in the existing function, have it default to `""` and set network_prefix differently depending on whether it is set or not.

Reply:
@ambroise-arm Thank you. Done